### PR TITLE
Add email when adherent register on citizen action

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: app/config/services/platform.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
             <tag name="security.voter"/>
         </service>
 
-        <!-- Citizen projects -->
+        <!-- Citizen projectss -->
```
Comment (thomasbisignani) [inline app/config/services/platform.xml]: Can you fix this typo on this branch?

File: src/Event/EventRegistrationCommandHandler.php [hunk 2]
```diff
@@ -40,11 +42,17 @@ public function handle(EventRegistrationCommand $command): void
...
-        $eventLink = $this->generateUrl('app_event_show', [
-            'slug' => $command->getEvent()->getSlug(),
-        ]);
-
-        $this->mailer->sendMessage(EventRegistrationConfirmationMessage::createFromRegistration($registration, $eventLink));
+        if ($command->getEvent() instanceof CitizenAction) {
+            $calendarExportLink = $this->generateUrl('app_event_export_ical', [
+                'slug' => $command->getEvent()->getSlug(),
+            ]);
+            $this->mailer->sendMessage(CitizenActionRegistrationConfirmationMessage::createFromRegistration($registration, $calendarExportLink));
```
Comment (B-Galati) [inline src/Event/EventRegistrationCommandHandler.php]: `sendMessage` can be done outside after the if statement

File: templates/citizen_action/show.html.twig [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{% extends 'events/show.html.twig' %}
+
+{% block page_title citizen_action.name %}
+{% block canonical_url url('app_citizen_action_show', { slug: citizen_action.slug }) %}
+
+{% block json_ld %}
+    {% set event = citizen_action %}
```
Comment (B-Galati) [inline templates/citizen_action/show.html.twig]: I am not sure. Is it possible to set the `event` variable outside of block ? Like we could avoid the same line 3 times.

File: templates/citizen_initiative/show.html.twig [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {% extends 'events/show.html.twig' %}
 
 {% block page_title initiative.name %}
-{% block canonical_url url('app_citizen_initiative_show', {'slug': initiative.slug}) %}
+{% block canonical_url url('app_citizen_initiative_show', { slug: initiative.slug }) %}
```
Comment (B-Galati) [inline templates/citizen_initiative/show.html.twig]: cs

Comment (thomasbisignani) [inline templates/citizen_initiative/show.html.twig]: Can you remove this whitespaces ?

**Pull request merged**